src/App.js

```diff
@@ -2,23 +2,27 @@ import React, { Component } from "react"
 import "./App.css"
 import firebase from "firebase"
 import StyledFirebaseAuth from "react-firebaseui/StyledFirebaseAuth"
+import Navigator from './components/routerComponent';
+
+// Your web app's Firebase configuration
+var firebaseConfig = {
```

Comment:
I think we should hide/protect this information, as it now is made public (which is something we don't want)

Reply:
Tried using GitHub secrets, but could not get it to work
Maybe someone else will have more succes

Reply:
"***The apiKey essentially just identifies your Firebase project on the Google servers. It is not a security risk for someone to know it. In fact, it is necessary for them to know it, in order for them to interact with your Firebase project.***"

So this doesn't matter

https://stackoverflow.com/questions/37482366/is-it-safe-to-expose-firebase-apikey-to-the-public